### DISCUSSION TITLE
How do I track all pipelines in a project ?

### DISCUSSION THREAD

**Discussion by shimi25** [mvisonneau/gitlab-ci-pipelines-exporter#826] [Q&A]
Hello, 

How do I track all CI pipelines in my GitLab server ?
I mean all pipeline on every branch (not only pipelines on master/main branches and tags) in the list of projects I've supplied  ?
How do I modify the 'gitlab-ci-pipelines-exporter.yml' file to achieve that ? 

Thanks !

Shimon.

**Comment by laimis9133** [comment]
Hey,
You can have a look at the configuration syntax [over here](https://github.com/mvisonneau/gitlab-ci-pipelines-exporter/blob/main/docs/configuration_syntax.md). The default value is behind a `project_defaults.pull.refs.branches.regexp: "^(?:main|master)$"`, you can modify it to match your branch namings. There are additional options that can help narrowing down exactly what you need.
